.gitignore

```diff
@@ -18,4 +18,7 @@ pnpm-debug.log*
 __coverage__/
 
 # Vercel output
-.vercel
\ No newline at end of file
+.vercel
+
+# Created by @astrojs/check
+/src/
```

Comment:
Huh, do you know why `@astrojs/check` is outputting stuff? Might be a bug.

Reply:
The file is `src/env.d.ts` which contain `/// <reference types="astro/client" />` so I would guess that `astro check` would run something similar to `astro sync` from where it's invoked which would generate this file at the root in our case.

But that's a guess so we may need to ask for a more definitive answer.

Reply:
This is indeed the case: [reference](https://discord.com/channels/830184174198718474/878341752799494154/1237151910385942691)